src/main/java/com/adobe/aem/compgenerator/utils/DialogUtils.java

```diff
@@ -300,11 +313,10 @@ private static void addBasicProperties(Element propertyNode, Property property)
      * @param property The {@link Property} object contains attributes
      */
     private static void addImagePropertyValues(Element imageNode, Property property) {
-        imageNode.setAttribute(Constants.PROPERTY_NAME, "./" + property.getField() + "/file");
-        imageNode.setAttribute(Constants.PROPERTY_CQ_MSM_LOCKABLE, "./" + property.getField() + "/file");
         imageNode.setAttribute("allowUpload", "{Boolean}false");
         imageNode.setAttribute("autoStart", "{Boolean}false");
         imageNode.setAttribute("class", "cq-droptarget");
+        imageNode.setAttribute("fileNameParameter", "./" + property.getField() + "/fileName");
```

Comment:
When we initially created this we purposely omitted the ability for an author to upload a file directly to a component (as opposed to referencing an asset in the DAM).  This helps promote asset reuse, asset organization and asset renditioning by forcing assets to all be in the DAM.  If we add this in, we should at least have it driven by a toggle so that it's not automatic.

Reply:
It's still got `allowUpload="{Boolean}false"` as the default, i.e., no upload allowed by default. I just added this `fileNameParameter` attribute to bring the list of defaults into line with the fields supported by latest image v2. In fact, the way image components were being generated under multifields before this change actually resulted in them ONLY allowing upload, because the default `allowUpload="{Boolean}false"` preference wasn't being included on them.

Reply:
ah k sounds good, thanks!